reqs/management/commands/import_reqs.py

```diff
@@ -93,6 +93,36 @@ def find_key(key_name, row):
     return keys[0]
 
 
+def handle_transposed_reqstatus(row):
+    """
+    There are a number of rows that have date content in the ``reqStatus``
+    field and non-date content in the ``policySunset`` field.
+
+    We're going to assume that this means they were transposed, and correct
+    those rows.
+    """
+    if "reqStatus" not in row:
```

Comment:
Sure; I left it out because it didn't come up with the current data, but it's probably worth adding.